cfgov/jinja2/v1/learn-page/index.html

```diff
@@ -226,7 +226,6 @@
 {% block content_sidebar scoped -%}
     {% if prototype %}
         {{ related_posts.render(global_dict.related_posts_settings) }}
-        {% include 'molecules/related-topics.html' %}
```

Comment:
yeah i haven't updated it since i found out they also have it. thanks for the reminder.